standard/ERCs/erc-6900.md

```diff
@@ -471,72 +470,55 @@ An account can have the same validation module installed more than once.
-- Revert if any address in `dependencies` does not support the interface at its matching index in the manifest's `dependencyInterfaceIds`, or if the two array lengths do not match, or if any of the dependencies are not already installed on the modular account.
+- the account MUST install all pre validation hooks required by the user and call `onInstall` with the user-provided data on the hook module to initialize the states.
+- the account MUST install all permission hooks required by the user and call `onInstall` with the user-provided data on the hook module to initialize the states.
+- the account MUST add all selectors required by the user that the validation can validate.
+- the account MUST set `isGlobal` and `isSignatureValidation` flags as required.
+- the account MUST call `onInstall` on the validation module to initialize the states.
+- the account MUST emit `ValidationInstalled` as defined in the interface for all installed validations.
 
-The function MUST record the manifest hash and dependencies that were used for the module's installation. Each dependency's record MUST also be updated to reflect that it has a new dependent. These records MUST be used to ensure calls to `uninstallModule` are comprehensive and undo all edited configuration state from installation. The mechanism by which these records are stored and validated is up to the implementation.
+During validation uninstallation, the account MUST correctly clear flags and other fields based on the incoming data provided by users.
 
-The function MUST store the module's permitted function selectors, permitted external calls, and whether it can spend the account's native tokens, to be able to validate calls to `executeFromModule` and `executeFromModuleExternal`.
+- the account MUST clear all flags for the validation function, including `isGlobal` and `isSignatureValidation`.
+- the account MUST remomve all hooks and clear hook module states by calling `onUninstall` with the user-provided data for each hook, including both pre validation hooks and permission hooks.
+- the account MUST remove all selectors that the validation function can validate.
+- the account MUST emit `ValidationUninstalled` as defined in the interface for all uninstalled validations.
 
-The function MUST parse through the execution functions, validation functions, and hooks in the manifest and add them to the modular account after resolving each `ManifestFunction` type.
+#### Execution and their installation /uninstallation
 
-- Each execution function selector MUST be added as a valid execution function on the modular account. If the function selector has already been added or matches the selector of a native function, the function SHOULD revert.
-- If a validation function is to be added to a selector that already has that type of validation function, the function SHOULD revert.
+An account can install any number of execution functions.
+An execution function selector MUST be unique to the account.
```

Comment:
```suggestion
An execution function selector MUST be unique in the account.
```
Mega-small clarification, because the wording implies this selector must only be present in this account, but we're trying to say the account cannot have multiple of the same selectors installed.